include/qpdf/QPDF.hh

```diff
@@ -239,6 +239,8 @@ class QPDF
     QPDFObjectHandle getTrailer();
     QPDF_DLL
     QPDFObjectHandle getRoot();
+    QPDF_DLL
+    std::map<QPDFObjGen, QPDFXRefEntry> const& getXRefTable() const;
```

Comment:
This has to be a non-const method that returns a copy, and it has to make sure it has been called after parse() so that the xref table is valid. I'm not adding any methods that return references to private members. I would probably accept the addition of this method as `std::map<QPDFObjGen, QPDFXrefEntry> getXRefTable()`.

Reply:
I've fixed the commit.
Thank you.